tests/testthat/apps/pws_ggplot2/tests/testthat/test-shinytest2.R

```diff
@@ -0,0 +1,257 @@
+library(shinytest2)
+
+# one app opens for all tests
+app <- AppDriver$new(name = "pws", height = 937, width = 1619)
+
+# tests plot click functionalities
+test_that("{shinytest2} plot_with_settings: click functionalities ggplot2", {
```

Comment:
All events in this tests are `app$set_inputs` which in my understanding change the binding directly so basically this test doesn't differ from `shiny::testServer`. In order to test it properly we need `app$click("#DOM ID")`. I guess thats because single click is easy to do, but click-and-drag-a-coursor not straightforward. If none of the brushing is possible by $click, then good idea is to replace this with shiny::testServer

Reply:
This issue has been created to tackle the server tests before moving to the UI tests issue created above: https://github.com/insightsengineering/teal.transform/issues/109